float32 x
float32 y
float32 type
float32 hist1
float32 hist2
float32 hist3
float32 hist4
float32 hist5
float32 hist6
float32 hist7
float32 hist8
float32 hist9
float32 hist10
float32 hist11
float32 hist12
float32 hist13
float32 hist14
float32 hist15
float32 hist16
float32 hist17
